Source/Classes/PINRemoteImageDownloadQueue.m

```diff
@@ -150,7 +150,7 @@ - (void)setQueuePriority:(PINRemoteImageManagerPriority)priority forTask:(NSURLS
                     break;
                     
                 default:
-                    NSAssert(NO, @"invalid priority: %d", priority);
+                    NSAssert(NO, @"invalid priority: %lu", priority);
```

Comment:
I think we need `%tu` here actually. `%lu` is only correct when building for 64bit targets. On 32bit targets `%lu` will still have the warning.

Source: https://twitter.com/gparker/status/377910611453046784?lang=en